java/code/src/com/redhat/rhn/common/hibernate/AnnotationRegistry.java

```diff
@@ -117,93 +162,139 @@ private AnnotationRegistry() {
-        ImageStore.class,
-        ImageStoreType.class,
-        DockerfileProfile.class,
-        KiwiProfile.class,
-        ImageProfile.class,
-        ProfileCustomDataValue.class,
-        DeltaImageInfo.class,
-        ImageFile.class,
-        ImageInfo.class,
-        ImageInfoCustomDataValue.class,
-        ImageOverview.class,
-        ImagePackage.class,
-        ImageRepoDigest.class,
-        VirtualHostManagerNodeInfo.class,
-        NotificationMessage.class,
-        UserNotification.class,
-        SCCRepository.class,
-        SCCSubscription.class,
-        SCCOrderItem.class,
-        ChannelTemplate.class,
-        SCCRepositoryAuth.class,
-        SCCRepositoryNoAuth.class,
-        SCCRepositoryBasicAuth.class,
-        SCCRepositoryTokenAuth.class,
-        SCCRepositoryCloudRmtAuth.class,
-        ContentProject.class,
-        ContentEnvironment.class,
-        ProjectSource.class,
-        SoftwareProjectSource.class,
-        ContentFilter.class,
-        ContentProjectFilter.class,
-        PackageFilter.class,
-        ErrataFilter.class,
-        ModuleFilter.class,
-        PtfFilter.class,
-        EnvironmentTarget.class,
-        SoftwareEnvironmentTarget.class,
-        ContentProjectHistoryEntry.class,
-        PackageExtraTagsKeys.class,
-        PackageProvides.class,
-        PackageRequires.class,
-        PackageRecommends.class,
-        PackageObsoletes.class,
-        PackageBreaks.class,
-        PackageSupplements.class,
-        PackageConflicts.class,
-        PackageSuggests.class,
-        PackagePreDepends.class,
-        PackageEnhances.class,
-        MinionRecurringAction.class,
-        GroupRecurringAction.class,
-        OrgRecurringAction.class,
-        MaintenanceSchedule.class,
-        MaintenanceCalendar.class,
-        SCCRegCacheItem.class,
-        AnsiblePath.class,
-        InventoryPath.class,
-        PlaybookPath.class,
-        Pillar.class,
-        CloudRmtHost.class,
-        PaygSshData.class,
-        PaygCredentialsProduct.class,
-        Task.class,
-        RecurringHighstate.class,
-        RecurringState.class,
-        RecurringConfigChannel.class,
-        RecurringInternalState.class,
-        InternalState.class,
-        PaygDimensionComputation.class,
-        PaygDimensionResult.class,
-        BaseCredentials.class,
-        CloudRMTCredentials.class,
-        RegistryCredentials.class,
-        ReportDBCredentials.class,
-        RHUICredentials.class,
-        SCCCredentials.class,
-        VHMCredentials.class,
-        ChannelSyncFlag.class,
-        ServerCoCoAttestationConfig.class,
-        ServerCoCoAttestationReport.class,
-        CoCoEnvironmentTypeConverter.class,
-        CoCoAttestationResult.class,
-        CoCoResultTypeConverter.class,
-        ServerAppStream.class,
-        AppStream.class,
-        AppStreamApi.class,
-        TokenChannelAppStream.class
+//            AvailableChannelsView.class,
```

Comment:
Could you avoid a giant diff hunk just for a commented line? And leaving commented code is a bad habit: should it be kept and fixed or removed?

Reply:
I can remove the comment, but the giant diff would remain since I order them in alphabetic order: I'll add a comment to be sure next class would be added in this way.